business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/model/KieWbDistribution.java

```diff
@@ -22,8 +22,7 @@
 public enum KieWbDistribution {
 
     KIE_WB,
```

Comment:
KIE_WB -> BUSINESS_CENTRAL to keep consistency.

Reply:
The reason why I didn't modify this was that kie-wb.jsp was not renamed - renaming it seemed to impact in many different places - however, to allow us to rename the profile I adjusted the tests to avoid it getting the app name from the system property (I hardcoded it to kie-wb.jsp). Let me know if it is fine.

Reply:
Yes, that's what I did in my PR to your branch too. It makes sense to hardcode it in a constant now (since we only have one "main" jsp file).